apis/v1alpha1/generator.yaml

```diff
@@ -433,6 +467,8 @@ resources:
 ignore:
   resource_names:
   - EksAnywhereSubscription
+  shape_names:
+    - Update
```

Comment:
Is this related to IdentityProviderConfig or Addons?

Reply:
This is related to identityProviderConfig.
[Associate](https://docs.aws.amazon.com/eks/latest/APIReference/API_AssociateIdentityProviderConfig.html) api returns `update` type. But [describe](https://docs.aws.amazon.com/cli/latest/reference/eks/describe-identity-provider-config.html) does not return that. So, essentially `update` turns out to be incomplete field which cant be updated once `associate` is done. e.g. `associate` sets one of the fields in `update` to be `inProgress`. But since `describe` does not have this, this field is never set to `active`. 

This `inPogress` string is not even part of statuses returned by `describe` api. ref: https://https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/eks/client/describe_identity_provider_config.html#describe-identity-provider-config

I did not find this to be useful to retain.

Reply:
I updated generator.yaml and added   - AssociateIdentityProviderConfigOutput.Update. This will only remove update field from identity-provider-config. and wont touch Addon